testing/src/test/java/io/stargate/it/http/docsapi/DocsHttpClient.java

```diff
@@ -0,0 +1,151 @@
+package io.stargate.it.http.docsapi;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import okhttp3.*;
+
+public class DocsHttpClient {
```

Comment:
Why not use io.stargate.it.http.RestUtils? That's why it's there.

Reply:
Yeaaaa I should - idk why I chose to do this